lib/util.js

```diff
@@ -314,6 +315,28 @@ function formatValue(ctx, value, recurseTimes) {
       keys.unshift('size');
     empty = value.size === 0;
     formatter = formatMap;
+  } else if (value instanceof ArrayBuffer) {
```

Comment:
I put it here because it adds a key for `.byteLength` but I can probably move it up and call `formatProperty()` directly, likewise for DataView.  Good idea / bad idea?